### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by aspala**
This PR is part of a campaign to fix a lot of typos on github!
You can see the progress on https://github.com/fixTypos/fix_typos/

https://github.com/client9/misspell

### CODE REVIEW

**Comment by hikaru-light** [comment]
Sorry, is this word "flavour" regarded as typo?
Let me know your opinion.

File: docs/_posts/2013-11-06-community-roundup-10.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ The fact that you can control when components are rendered is a very important c
...
+> Eerily similar, no? Maybe Facebook was inspired by Fruit Machine (after all, we got there first), but more likely, it just shows that this is a pretty decent way to solve the problem, and great minds think alike. We're graduating to a third phase in the evolution of web best practice - from intermingling of markup, style and behavior, through a phase in which those concerns became ever more separated and encapsulated, and finally to a model where we can do that separation at a component level. Developments like Web Components show the direction the web community is moving, and frameworks like React and Fruit Machine are in fact not a lot more than polyfills for that promised behavior to come.
```
Comment (nhunzaker) [inline docs/_posts/2013-11-06-community-roundup-10.md]: For other reviewers, the change here is from "behaviour" to "behavior"

File: fixtures/dom/src/components/fixtures/text-inputs/index.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ class TextInputFixtures extends React.Component {
     return (
       <FixtureSet
         title="Inputs"
-        description="Common behavior across controled and uncontrolled inputs"
+        description="Common behavior across controlled and uncontrolled inputs"
```
Comment (nhunzaker) [inline fixtures/dom/src/components/fixtures/text-inputs/index.js]: Hey! This typo is me! Thanks!

File: scripts/fiber/tests-passing.txt [hunk 3]
```diff
@@ -105,8 +105,8 @@ src/isomorphic/classic/class/__tests__/create-react-class-integration-test.js
 * should warn on invalid prop types
 * should warn on invalid context types
 * should throw on invalid child context types
-* should warn when mispelling shouldComponentUpdate
-* should warn when mispelling componentWillReceiveProps
+* should warn when misspelling shouldComponentUpdate
+* should warn when misspelling componentWillReceiveProps
```
Comment (nhunzaker) [inline scripts/fiber/tests-passing.txt]: Ironic

File: docs/_posts/2013-11-06-community-roundup-10.md (hunk 1 above)
Comment (alexeyraspopov) [inline docs/_posts/2013-11-06-community-roundup-10.md]: 🇬🇧

File: scripts/fiber/tests-passing.txt (hunk 3 above)
Comment (hikaru-light) [inline scripts/fiber/tests-passing.txt]: I thought so, too.

File: docs/_css/bourbon/css3/_flex-box.scss [hunk 4]
```diff
@@ -58,7 +58,7 @@
 }
 
 // CSS3 Flexible Box Model and property defaults
-// Unified attributes for 2009, 2011, and 2012 flavours.
+// Unified attributes for 2009, 2011, and 2012 flavors.
```
Comment (hikaru-light) [inline docs/_css/bourbon/css3/_flex-box.scss]: Yes, I was just wondering.

**Comment by aspala** [comment]
I reverted the typo fixes within the flexbox dep.

Thank you for doing such a great job!

**Comment by aweary** [comment]
@aspala can you resolve the conflict(s) when you get a chance? We should be good to merge after that. Thanks!

**Pull request closed without merging**